src/3.2/Dockerfile

```diff
@@ -4,8 +4,8 @@ RUN apk add --no-cache --quiet \
     bash \
     curl
 
-ENV NEO4J_SHA256 %%NEO4J_SHA%%
-ENV NEO4J_TARBALL %%NEO4J_TARBALL%%
+ENV NEO4J_SHA256 %%NEO4J_SHA%% \
+    NEO4J_TARBALL %%NEO4J_TARBALL%%
```

Comment:
Reference: https://docs.docker.com/engine/reference/builder/#env

Use equal signs instead of space to fix this problem:

```
ENV NEO4J_SHA256=%%NEO4J_SHA%% \
    NEO4J_TARBALL=%%NEO4J_TARBALL%%
```